jose/Utils.ml

```diff
@@ -20,6 +20,11 @@ module RResult = struct
     | Ok a, Ok b, Ok c, Ok d, Ok e, Ok f, Ok g, Ok h ->
         Ok (a, b, c, d, e, f, g, h)
     | _ -> Error (`Msg "all 8 was not Ok")
+
+  let get_exn = function
+    | Ok a -> a
+    | Error (`Msg m) -> raise (Invalid_argument m)
+    | Error _ -> raise Not_found
```

Comment:
I'd `failwith "get_exn"`. `Not_found`s alone are a little hard to track down.

Reply:
I plan on removing this before merging the PR